src/components/pill-selector-dropdown/PillSelector.js

```diff
@@ -207,8 +214,10 @@ class PillSelector extends React.Component<Props, State> {
                         onBlur={this.resetSelectedIndex}
                         ref={this.hiddenRef}
                         tabIndex={-1}
+                        data-testid="pill-selection-helper"
                     />
                     <textarea
+                        {...ariaAttrs}
```

Comment:
Unfortunately `PillSelector` isn't currently using the `TextArea` component, so I had to do this separately. TODO: Move to `<TextArea>`.